### PR TITLE
Added changes for a new exercise type walking 

### PR DESCRIPTION
**Pull request by ruchika512**
Added changes for a new exercise type walking in frontend, activity tracking and analytics services
image
image
image
image
image

### CODE REVIEW

**Review by HarshiSharma — COMMENTED** [review]
LGTM. Just wanted to ask the colour you have used is from accessible palette?

File: frontend/src/components/journal.js
Comment (HarshiSharma) [inline frontend/src/components/journal.js]: can we change this to:
`const order = ['Running', 'Swimming', 'Cycling', 'Walking', 'Gym', 'Other'];` just so that all exercise with stats appear first.

Comment (ruchika512) [reply frontend/src/components/journal.js]: Updated the ordering for Journal page 
image

**Comment by ruchika512** [comment]
> LGTM. Just wanted to ask the colour you have used is from accessible palette?

Updated the color to an accessible as per WCAG AAA

**Review by VathsalaAchar — APPROVED** [review]
Nice work

**Comment by VathsalaAchar** [comment]
Waiting on review and approval by @HarshiSharma before merging in

**Review by HarshiSharma — APPROVED** [review]
Nicely Done!

**Pull request merged**
